app/views/checkout/payment.scala.html

```diff
@@ -1,95 +1,119 @@
 @(form: Form[model.SubscriptionData], userIsSignedIn: Boolean)(implicit request: RequestHeader)
 
-@import routes.Checkout.renderCheckout
-
 @import model.JsVars
+@import routes.Checkout.renderCheckout
 @import configuration.Config.Identity._
 
 @readonly = @{ if (userIsSignedIn) "readonly" else "" }
 
-@main(title = "Checkout | Subscriptions | The Guardian",
-      jsVars = JsVars.default.copy(userIsSignedIn = userIsSignedIn)) {
+@main(
+    title = "Checkout | Subscriptions | The Guardian",
+    jsVars = JsVars.default.copy(userIsSignedIn = userIsSignedIn),
+    bodyClasses = List("is-wide")
+) {
 
 <div class="gs-container">
-    @fragments.page.header("Checkout")
-
-    <div>
-    @if(userIsSignedIn) {
-        <h3>Not you?</h3>
-        <a class="action" href="@idWebAppSignOutUrl(renderCheckout.toString)">Sign out</a>
-    } else {
-        <h3>Already have a Guardian account?</h3>
-        <a class="action" href="@webAppSigninUrl(renderCheckout.toString)">Sign in</a>
-    }
-    </div>
 
-    <div class="row row--items-2">
-        <div class="row__item row__item--boost-2">
+    <div class="checkout-container">
+
+        <div class="checkout-header">
+            <h1 class="checkout-header__title">Checkout</h1>
+        </div>
+
+        <div class="checkout-container__sidebar">
+            @fragments.checkout.basketPreview()
+
+            <div class="js-checkout-smallprint is-hidden">
+                @fragments.checkout.notice("Advanced Notice") {
+                    <p>The details of your Direct Debit instruction including payment schedule, due date, frequency and amount will be sent to you within three working days. All the normal Direct Debit safeguards and guarantees apply.</p>
+                }
+                @fragments.checkout.notice("Money Back Guarantee") {
+                    <p>If you wish to cancel your subscription, we will send you a refund of the unexpired part of your subscription.</p>
+                }
+                @fragments.checkout.notice("Direct Debit") {
+                    @* TODO: Refactor this markup *@
+                    <ul class="u-unstyled-list">
+                        <li>The Guardian</li>
```

Comment:
Not that I know of, this is details about Direct Debit requirements. Real copy, the `TODO` is just a reminder to me to clean up the markup, it doesn't need to be a list.